vector/src/main/java/fr/gouv/tchap/activity/TchapRoomCreationActivity.java

```diff
@@ -130,6 +130,13 @@ public void initUiAndData() {
         switchPublicPrivateRoom.setChecked(false);
         mRoomParams.visibility = RoomDirectoryVisibility.DIRECTORY_VISIBILITY_PRIVATE;
         mRoomParams.preset = CreateRoomParams.PRESET_PRIVATE_CHAT;
+        // Hide the encrypted messages sent before the member is invited.
+        mRoomParams.setHistoryVisibility(RoomState.HISTORY_VISIBILITY_INVITED);
+
+        // A Tchap room member must be moderator to invite
+        final Map<String, Object> powerLevelContentOverride = new HashMap<>();
```

Comment:
there is no need to introduce a local variable, you can directly affect `mRoomParams.powerLevelContentOverride`